### ISSUE TITLE
Perl 5.38 method is ignored

### ISSUE THREAD

**Issue by brainbuz** [pjcj/Devel--Cover#330]
method  enabled with the 'class' feature of Perl 5.38 is not included in devel cover stats.

I'm using Perl 5.38 for a project I'm going to release as an ACME module and Devel::Cover is missing the method statements.

The public repo if you want to run coverage against my code is https://bitbucket.org/brainbuz/lottery

I put the coverage report in a snippet: https://bitbucket.org/brainbuz/workspace/snippets/q7neq5

**Comment by pjcj** [comment]
Yes, I've not done anything special for any of the new syntax.  Thanks for the report - I'll have to try to carve out some time to investigate further.

**Comment by tessarin** [comment]
@pjcj Do you have any updates to this? I'm also using core classes everywhere now, and I really miss Devel::Cover parsing methods. If there are any guidelines for contributing, I’d be happy to assist.

**Comment by sidyll** [comment]
Is this module still maintained? @pjcj? It's been more than one year without anything here.

**Comment by pjcj** [comment]
@sidyll I suppose that depends on what you mean by maintained.  Every year for over 20 years the module has worked when the latest release of perl has been released.  That should continue to be true when 5.42.0 is released in the next few days or weeks.

And I'll generally make sure pull requests are handled.  But, as you can probably see, I don't have a lot of time for dealing with other matters, and that often includes new perl features.

@tessarin There are two areas which may need looking into.  The first is checking the opcodes and storing the coverage information as the code is run.  I'm not sure whether there are any new opcodes or parts of the optree which need to ba handled.  This code is in `Cover.xs`. 

The second is in storing the coverage data in the coverage DB.  This may involve looking through the optree for new codes and storing the coverage associated with them.  This code is in `lib/Devel/Cover.pm`.

We'll also need tests to exercise all the new features we want to cover.
